x/staking/keeper/common_test.go

```diff
@@ -33,6 +28,7 @@ func createTestInput(t *testing.T) (*codec.LegacyAmino, *simapp.SimApp, sdk.Cont
 		app.AccountKeeper,
 		app.BankKeeper,
 		app.GetSubspace(types.ModuleName),
+		types.DefaultConfig(),
```

Comment:
shouldn't we use `sdk.NewIntFromBigInt(new(big.Int).Exp(big.NewInt(10), big.NewInt(18), nil))` here because of the deleted line above?